packages/telestion-client-cli/src/lib/get-config.js

```diff
@@ -13,7 +13,7 @@ async function getConfig() {
 
 	if (config === null) {
 		throw new Error(
-			'No config file found. Expected to find .telestionrc, telestion.config.js, or something similar. ' +
+			'No config file found. Expected to find .telestionrc, telestion.get-config.js, or something similar. ' +
```

Comment:
```suggestion
			'No config file found. Expected to find .telestionrc, telestion.config.js, or something similar. ' +
```
I think, the replacer has hit something not meanted to be changed :smile: